components/views/faq/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react'
+import { StyledFAQ } from './FAQ.styles'
+import {
+    Accordion,
+    AccordionDetails,
+    AccordionSummary,
+    Typography
+} from '@mui/material'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+
+import DummyFAQs from './dummydata'
+
+const FAQPage = () => {
+    const [expanded, setExpanded] = useState<string | false>(false)
+
+    const handleChange =
+        (panel: string) =>
+        (event: React.SyntheticEvent, isExpanded: boolean) => {
+            setExpanded(isExpanded ? panel : false)
+        }
+
+    return (
+        <StyledFAQ>
+            <div className="FAQ-hero">
+                <h1>Title</h1>
+                <h2>Subtext</h2>
```

Comment:
Are these 2 lines going to be added to the schema so the client can change it as needed ?